spec/models/category_spec.rb

```diff
@@ -17,18 +19,17 @@
     end
 
     it "must return 6 videos at most" do
-      stargate = Fabricate.times(7,:video, category: category)
+      Fabricate.times(7,:video, category: category)
       expect(category.recent_videos.count).to eq(6)
     end
 
     it "must be ordered by created_at ascending(last video first)" do
-      stargate_seasons = 7
-      seasons_array = []
-      stargate_seasons.times do |season|
-        stargate = Fabricate(:video, category: category)
-        seasons_array << stargate
+      videos_array = []
+      7.times do |season|
+        video = Fabricate(:video, category: category)
+        videos_array << video

```

Comment:
The spec says it must be ordered by created_at. You do not need to create a mass of 7 videos. Create like 3 videos and just assert on exact array. like eq [video1, video2, video3].

Your specs should focus on unique aspicts, in previous you already tested the amount of 6, no need to build the longer array now anyway.